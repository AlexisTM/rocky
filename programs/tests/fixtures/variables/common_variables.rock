a variable is 1
the variable is 2
an variable is 3
my variable is 4
your variable is 5

the variables are 6
my variables are 7
your variables are 8

say a variable
say the variable
say an variable
say my variable
say your variable

say the variables
say my variables
say your variables

say "expressions"
say a variable with the variable
say my variable with your variable

say "case insensitivity"
THE TIME is 1
say THE TIME
say THE time
say The Time
say tHe tImE
say the Time
say the time

THE time is 2
say THE TIME
say THE time
say The Time
say tHe tImE
say the Time
say the time

The Time is 3
say THE TIME
say THE time
say The Time
say tHe tImE
say the Time
say the time

tHe tImE is 4
say THE TIME
say THE time
say The Time
say tHe tImE
say the Time
say the time

the Time is 5
say THE TIME
say THE time
say The Time
say tHe tImE
say the Time
say the time

the time is 6
say THE TIME
say THE time
say The Time
say tHe tImE
say the Time
say the time

the times are 7
say the times
the times aRE 8
say the times
