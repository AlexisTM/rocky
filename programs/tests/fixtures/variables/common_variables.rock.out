1
2
3
4
5
6
7
8
expressions
3
9
case insensitivity
1
1
1
1
1
1
2
2
2
2
2
2
3
3
3
3
3
3
4
4
4
4
4
4
5
5
5
5
5
5
6
6
6
6
6
6
7
8
